specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2022-09-08-preview/managedccf.json

```diff
@@ -0,0 +1,555 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-08-preview",
+    "title": "ConfidentialLedger",
+    "description": "Microsoft Azure Confidential Compute Ledger Managed CCF Control Plane REST API version 2022-09-08-preview."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ConfidentialLedger/managedCCF/{appName}": {
```

Comment:
>managedCCF [](http://example.com/codeflow?start=111&length=10)

Recommend non-abbreviated resource type name. Also suggest updating descriptions. This comment applies throughout the PR.

Reply:
This is the marketing name and should be well known, I don't think we can expand it since it'll be way too long as a resourcetype.
But we can expand mccf into managedccf if needed

Reply:
since its a collection this needs to be plural

Reply:
I have placed the ARMChangesRequested label just for making this change. The rest is fine. Please drop the label once you have addressed this.

Reply:
thanks a lot, i have made the change

Reply:
Not fixed. The resource name must be plural since it is a collection of resources:  managedccfs/{name}

Reply:
@raosuhas is this mandatory? We already have checked in the same resourcetype in the private repo and this change will cause many breaking changes on our side and we are close to a preview release. Also, "ManagedCCFs" is not syntactically correct on our case. 
Adding @msftsettiy and @sidmore to have also their thoughts on this

Reply:
@raosuhas / @rkmanda / @mentat9  I've renamed "ManagedCCF" into "ManagedCCFs" now. Appreciate if we can merge the PR